Core/Core/Planner/APICalendarEvent.swift

```diff
@@ -106,22 +106,22 @@ public struct GetCalendarEventsRequest: APIRequestable {
     public let contexts: [Context]?
     public let startDate: Date?
     public let endDate: Date?
+    private let calendar: Calendar
+    private let timeZone: TimeZone
     public let type: CalendarEventType
     public let perPage: Int
     public let include: [Include]
     public let allEvents: Bool?
     public let userID: String?
     public let importantDates: Bool?
-    public static let dateFormatter: DateFormatter = {
-        let df = DateFormatter()
-        df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss'Z'"
-        return df
-    }()
+    public var useExtendedPercentEncoding: Bool = true
```

Comment:
We shouldn't allow this variable to be settable because it will break the request if set to false.

Reply:
Changed it to a computed property.